erpnext/patches/v13_0/custom_field_tax_collectable_and_taxable_amount_in_sales_invoice.py

```diff
@@ -0,0 +1,30 @@
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields    
+
+def execute():
+	enabled = frappe.get_all("TaxJar Configuration", filters={"enabled": True})
+
+	if not enabled:
+		return
+	
+	custom_fields = {
+		'Quotation Item': [
+			dict(fieldname='tax_collectable', fieldtype='Currency', insert_after='net_amount',
+				label='Tax Collectable', read_only=1, options='currency'),
+			dict(fieldname='taxable_amount', fieldtype='Currency', insert_after='tax_collectable',
+				label='Taxable Amount', read_only=1, options='currency')
+		],
+		'Sales Order Item': [
+			dict(fieldname='tax_collectable', fieldtype='Currency', insert_after='net_amount',
+				label='Tax Collectable', read_only=1, options='currency'),
+			dict(fieldname='taxable_amount', fieldtype='Currency', insert_after='tax_collectable',
+				label='Taxable Amount', read_only=1, options='currency')
+		],
+		'Sales Invoice Item': [
+            dict(fieldname='tax_collectable', fieldtype='Currency', insert_after='net_amount',
+				label='Tax Collectable', read_only=1, options='currency'),
+			dict(fieldname='taxable_amount', fieldtype='Currency', insert_after='tax_collectable',
+				label='Taxable Amount', read_only=1, options='currency')
+		]
+	}
+	create_custom_fields(custom_fields, update=True)
```

Comment:
check if the custom fields already exist before creating

Reply:
create_custom_field itself check field exist or not